### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sashahilton00**
Updated the dependencies to use newer version of farmhash and the pure javascript version of toobusy, which enables ringpop to run on the latest stable node (v4.4.7) whilst also remaining compatible with node v0.10.41.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,10 +32,10 @@
   "dependencies": {
     "body": "^5.0.0",
     "error": "^5.0.0",
-    "farmhash": "^0.2.0",
+    "farmhash": "^1.1.0",
     "metrics": "^0.1.8",
     "node-uuid": "^1.4.3",
-    "toobusy": "^0.2.4",
+    "toobusy-js": "^0.5.0",
```
Comment (Raynos) [inline package.json]: You need to find the place where `toobusy` is required, and replace it with `require('toobusy-js')`

**Comment by sashahilton00** [comment]
I guess that's what I get for pushing at 2AM ;) Fixed it, it seems to only be used in lag_sampler.js.

**Comment by sashahilton00** [comment]
ping @Raynos

**Comment by Raynos** [comment]
lgtm. You should get someone on the ringpop team to merge. cc @dansimau @thanodnl

**Comment by thanodnl** [comment]
@sashahilton00 Thanks for taking the time to create this patch. Every thing looks good to me as well. We will run some internal tests on this tomorrow to measure the impact and if things work we will incorporate these changes in the next release.

Before we are able to merge your changes we need you to agree to our Contributor License Agreement. Can you please have a look at http://t.uber.com/cla

Lastly, would you mind if we make some additional changes here to run TravisCI on node v4.4.7 as well? This makes sure that we notice when we make changes that break ringpop-node from running on this version of node.

**Comment by sashahilton00** [comment]
@thanodnl no problem. Have filled out that CLA, you guys should just change that to 'I certify that I will only submit code that is my own, and Uber can do whatever it wants with it' :) Anyway, feel free to make any Travis changes necessary, it's your server after all. Out of interest, are you guys going to be upgrading from node v0.10 anytime soon, given that io.js and node.js have been merged?

**Comment by thanodnl** [comment]
I'm closing this PR and continue working on #290 since I could not make changes on this branch, and github didn't let me change the branch for this PR.

Please subscirbe to #290 if you want to follow the progress.

**Pull request closed without merging**